package.json

```diff
@@ -134,5 +134,7 @@
   "eslintConfig": {
     "extends": "react-app"
   },
-  "dependencies": {}
+  "dependencies": {
+    "faker": "^4.1.0"
```

Comment:
We should probably move this to `devDependencies`.

This can be accomplished by:

```
npm uninstall faker
npm install faker --dev
```